### PR TITLE
Added option to configure the disk sizes

### PR DESCRIPTION
**Pull request by ronniel1**

### CODE REVIEW

**Comment by tsorya** [comment]
@ronniel1 can u rebase it?

**Comment by tsorya** [comment]
will reopen

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,8 @@ ISO := $(or $(ISO), "")
 NUM_MASTERS :=  $(or $(NUM_MASTERS),3)
 WORKER_MEMORY ?= 8892
 MASTER_MEMORY ?= 16984
+WORKER_DISK ?= 21474836480
+MASTER_DISK ?= 21474836480
```
Comment (masayag) [inline Makefile:9]: It will be useful to add those to the README file.

**Pull request closed without merging**